.dockerignore

```diff
@@ -0,0 +1,7 @@
+.venv
+.che__/
+*.log
+*.git
+*.gitignoreivenv
```

Comment:
what is going on with these ignore entries?  "che", "gitignoreivenv", ... what?

Also shouldn't eg `embeddings`, `extensions`, etc. be ignored here? (ie anything that has a separate bind to forward it in)

Reply:
.che__ is Eclipse env.
gitignoreivenv is a typo ... should be gitignore and .venv...

Reply:
fixed